src/hermes_claude_agent_sdk/parity/v4_fixture_materializer.py

```
@@ -0,0 +1,347 @@
+        surfaces = _BASE_SURFACES + tuple(_SURFACES_BY_MECHANISM[item["mechanism_class"]])
+        if child_ids:
+            surfaces += ("child",)
+        if item["path_policy"] == "host_denial_local_recovery":
+            surfaces += ("delivery_boundary",)
+        approval = item["mechanism_class"] == "host_tool_pdr" or item["path_policy"] == "host_denial_local_recovery"
+        sequence = ("deny", "safe_recovery") if approval and path == "positive" and "recovery" in item["mandatory_paths"] else (("deny",) if approval else ())
+        expected = V4HostSurfaceExpectations(
+            tuple(_TOOLS_BY_MECHANISM[item["mechanism_class"]]), tuple(dict.fromkeys(surfaces)),
+            "deny" if approval else "not_required", sequence, len(bindings),
+            tuple(binding[2] for binding in bindings), child_ids, label,
+            1 if path == "positive" else 0, False,
```

Ruff (0.16.3)

[warning] Boolean positional value in function call

(FBT003)